git.adoc

```diff
@@ -154,30 +154,34 @@ $ source ~/.bashrc
 +
 [source,terminal]
 ----
-$ git fetch upstream
+$ git fetch --all
 ----
 +
 [source,terminal]
 ----
 $ git rebase upstream/main
 ----
 +
-If you are working on an enterprise branch for release notes:
+If you are working on an enterprise branch for release notes (replace `$VERSION` with the version you are working on):
 +
 [source,terminal]
 ----
-$ git rebase upstream/enterprise-4.x
+$ git rebase upstream/enterprise-$VERSION
 ----
 
 == Git some
 
 === Be aware of open files when switching branches!
 
-When you switch branches, the files in your editor will remain open. Be careful when closing them. If you inadvertently save the file, you run the risk of accidentally adding the file to the wrong branch.
+When you switch branches, the files in your editor will remain open.
```

Comment:
[One sentence per line](https://opensource.com/article/19/4/write-git#one-sentence-per-line) FTW!

Reply:
Yes - I found this really interesting from your git session. I really need to start doing this. It's hard to break out of the regular word processing flow though.